docs/services/slam/cartographer/_index.md

```diff
@@ -272,17 +338,48 @@ See [Attributes](#attributes) for details.
 }
 ```
 
-{{% /expand%}}
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Start a mapping session
+
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+   On the cartographer panel, you can start a mapping session.
 
-#### Localize only
+   When you start a mapping session:
 
-In this mode, the cartographer-module on your robot executes the Cartographer algorithm itself locally.
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+     Because Cartographer's algorithm is CPU-intensive when creating or updating a map, in this mode the `cartographer` module on your robot acts as a stub.
+   - Viam's [Data Capture](/services/data/configure-data-capture/) continuously monitors and captures your robot's sensor data while the robot is running.
+   - Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the new map based on the existing map.
+     Once you click **End session**, the map is uploaded to the cloud and visible on your **Location** page under **SLAM library**.
 
-The configuration is similar to the configuration for [updating an existing map](#update-existing-map), except instead of setting a `Data capture rate (Hz)` on the camera and movement sensor, you set a `Data polling rate (Hz)` on them, since the `cartographer-module` on your robot will poll the live LiDAR and IMU directly at these rates instead of the data being sent to the cloud.
+   Enter a name for your new map and click **Start session**.
+   Wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
 
-{{%expand "Click to view an example JSON configuration for localizing only" %}}
+{{% alert title="Info" color="info" %}}
+
+Cartographer may take several minutes to find your robot's position on the existing map.
+In the meantime, your robot will show up at the map's origin (with the `(x,y)` coordinates `(0,0)`).
+
+You may not (?) want to move your robot until it has finished localizing correctly.
```

Comment:
@EshaMaharishi why would a user not want to do that? What happens if they do, does that lead to errors?

Reply:
Actually, I'm not sure if it will lead to errors. I had thought the scans might get applied to the wrong part of the map if you start moving, but I just realized that that would be true even if you weren't moving. So, I think we could just remove this line.